src/test/java/nextstep/subway/acceptance/PathAcceptanceTest.java

```diff
@@ -142,6 +166,153 @@ void longDistanceFare() {
+     */
+    @DisplayName("구간 길이가 10km 이내이고 여러 추가 요금 노선이 포함되었다면, 총 지하철 요금은 (기본 요금 + 가장 큰 노선 추가 요금) 이다.")
+    @Test
+    void basicDistanceWithExtraFares() {
+        // when
+        ExtractableResponse<Response> response = 두_역의_최단_거리_경로_조회를_요청(서울역, 동대문역);
+
+        // then
+        assertThat(response.jsonPath().getInt("fare")).isEqualTo(1_650);
+    }
+
+    /**
+     * When 출발역에서 도착역까지의 구간 길이가 10km 초과 50km 이내이고, 추가 요금이 있는 노선을 이용하면
+     * Then 총 지하철 요금은 (기본 요금 + 추가 요금(5km 당 100원) + 노선 추가 요금) 이다.
+     */
+    @DisplayName("구간 길이가 10km 초과 50km 이내이고 여러 추가 요금 노선이 포함되었다면, 총 지하철 요금은 (기본 요금 + 추가 요금(5km 당 100원) + 가장 큰 노선 추가 요금) 이다.")
+    @Test
+    void middleDistanceWithExtraFare() {
+        // when
+        ExtractableResponse<Response> response = 두_역의_최단_거리_경로_조회를_요청(성수역, 공덕역);
+
+        // then
+        assertThat(response.jsonPath().getInt("fare")).isEqualTo(2_250);
+    }
+
+    /**
+     * When 출발역에서 도착역까지의 구간 길이가 10km 초과 50km 이내이고, 추가 요금이 있는 여러 노선을 이용하면
+     * Then 총 지하철 요금은 (기본 요금 + 추가 요금(5km 당 100원) + 가장 큰 노선 추가 요금) 이다.
+     */
+    @DisplayName("구간 길이가 10km 초과 50km 이내이고 추가 요금 노선이 포함되었다면, 총 지하철 요금은 (기본 요금 + 추가 요금(5km 당 100원) + 노선 추가 요금) 이다.")
+    @Test
+    void middleDistanceWithExtraFares() {
+        // when
+        ExtractableResponse<Response> response = 두_역의_최단_거리_경로_조회를_요청(성수역, 서울역);
+
+        // then
+        assertThat(response.jsonPath().getInt("fare")).isEqualTo(2_850);
+    }
+
+    /**
+     * When 출발역에서 도착역까지의 구간 길이가 50km 초과이고, 추가 요금이 있는 노선을 이용하면
+     * Then 총 지하철 요금은 (기본 요금 + 추가 요금(8km 당 100원) + 노선 추가 요금) 이다.
+     */
+    @DisplayName("구간 길이가 50km 초과이고 추가 요금 노선이 포함되었다면, 총 지하철 요금은 (기본 요금 + 추가 요금(8km 당 100원) + 노선 추가 요금) 이다.")
+    @Test
+    void longDistanceWithExtraFare() {
+        // when
+        ExtractableResponse<Response> response = 두_역의_최단_거리_경로_조회를_요청(강남역, 공덕역);
+
+        // then
+        assertThat(response.jsonPath().getInt("fare")).isEqualTo(2_350);
+    }
+
+    /**
+     * When 출발역에서 도착역까지의 구간 길이가 50km 초과이고, 추가 요금이 있는 여러 노선을 이용하면
+     * Then 총 지하철 요금은 (기본 요금 + 추가 요금(8km 당 100원) + 가장 큰 노선 추가 요금) 이다.
+     */
+    @DisplayName("구간 길이가 50km 초과이고 여러 추가 요금 노선이 포함되었다면, 총 지하철 요금은 (기본 요금 + 추가 요금(8km 당 100원) + 가장 큰 노선 추가 요금) 이다.")
+    @Test
+    void longDistanceWithExtraFares() {
+        // when
+        ExtractableResponse<Response> response = 두_역의_최단_거리_경로_조회를_요청(강남역, 동대문역);
+
+        // then
+        assertThat(response.jsonPath().getInt("fare")).isEqualTo(2_950);
+    }
+
+    /**
+     * Given 성인 사용자가 로그인을 요청하고
+     * When 지하철 경로 조회를 요청한 로그인 사용자가 성인이라면
+     * Then 지하철 요금은 할인되지 않은 요금이다.
+     */
+    @DisplayName("로그인 사용자가 성인이라면, 지하철 요금은 할인되지 않은 요금이다.")
+    @Test
+    void findPathByAdultMember() {
+        // given
+        String accessToken = 베어러_인증_로그인_요청("adult@email.com", "password").jsonPath().getString("accessToken");
+
+        // when
+        ExtractableResponse<Response> response = 로그인_사용자가_두_역의_최단_거리_경로_조회를_요청(교대역, 강남역, accessToken);
```

Comment:
```suggestion
        var accessToken = 베어러_인증_로그인_요청("adult@email.com", "password").jsonPath().getString("accessToken");

        // when
        var response = 로그인_사용자가_두_역의_최단_거리_경로_조회를_요청(교대역, 강남역, accessToken);
```

저는 인수 테스트를 짤 때 더 간결하게 만들기 위하여 타입 추론을 사용하긴 합니다. 😊